### ISSUE TITLE
Better support for terminal

### ISSUE THREAD

**Issue by pomber** [code-hike/codehike#206]
Most docs start with some commands you need to run on a terminal. 

For example, one common use case for web projects is to show options for npm/yarn/pnpm. Maybe code hike should make that use case easier.

There's also the styling of commands and output. Here the important part is `yarn start`, the other lines are extra context:

````md
```terminal
$ yarn start
[INFO] Starting the development server...
[SUCCESS] Website is running at: http://localhost:3000/
```
````

**Comment by DanielFGray** [comment]
> show options for npm/yarn/pnpm

i've been wondering if CH can do this a la docusaurus, and if this could open the door to more "choose your adventure" kinds of features

**Comment by pomber** [comment]
> i've been wondering if CH can do this a la docusaurus, and if this could open the door to more "choose your adventure" kinds of features

Yes, it's doable right now. You can do something similar to this [language switcher example](https://codehike.org/docs/code/language-switcher). Since you own the `<Code>` component you can implement it any way you want.
